cicd/pipelines/authenticating-pipelines-and-tasks-using-secrets.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="authenticating-pipelines-and-tasks-using-secrets"]
+= Authenticating pipelines and tasks using secrets
+include::_attributes/common-attributes.adoc[]
+:context: authenticating-pipelines-and-tasks-using-secrets
+
+toc::[]
+
+For authenticating to GitHub, GitLab, or Docker in your Pipeline and Task, you can use secrets. Secrets are sensitive pieces of data that you can securely store in your project and access when needed. To use secrets for authentication, use the following recommended approaches:
```

Comment:
"or Docker" should probably be "or Container Image Registry" to be a bit more generic 👼🏼